tests/e2e/expected_results/slsa-verifier/slsa-verifier_cue_PASS.json

```diff
@@ -1768,6 +1768,18 @@
                     ],
                     "result_type": "PASSED"
                 },
+                {
+                    "check_id": "mcn_provenance_witness_level_one_1",
+                    "check_description": "Check whether the target has a level-1 witness provenance.",
+                    "slsa_requirements": [
+                        "Provenance - Available - SLSA Level 1",
+                        "Provenance content - Identifies build instructions - SLSA Level 1",
+                        "Provenance content - Identifies artifacts - SLSA Level 1",
+                        "Provenance content - Identifies builder - SLSA Level 1"
+                    ],
+                    "justification": [],
+                    "result_type": "PASSED"
```

Comment:
Same comment here.

Reply:
Fixed in d234c2736a370d13320b0b7338ba461224f2a31b.